$MirOS: ports/databases/postgresql/patches/patch-src_Makefile_shlib,v 1.1.7.1 2005/03/18 15:44:16 tg Exp $
$OpenBSD: patch-src_Makefile_shlib,v 1.2 2002/12/17 16:38:54 naddy Exp $
--- src/Makefile.shlib.orig	Tue Sep 19 17:35:44 2006
+++ src/Makefile.shlib	Thu Feb 22 20:25:47 2007
@@ -118,12 +118,7 @@ ifeq ($(PORTNAME), darwin)
 endif
 
 ifeq ($(PORTNAME), openbsd)
-  ifdef ELF_SYSTEM
-    LINK.shared		= $(COMPILER) -shared -Wl,-x,-soname,$(soname)
-    SHLIB_LINK		+= -lc
-  else
-    LINK.shared		= $(LD) -x -Bshareable -Bforcearchive
-  endif
+  LINK.shared		= $(COMPILER) -shared -Wl,-x,-soname,$(soname)
 endif
 
 ifeq ($(PORTNAME), bsdi)
